conf/logback.xml

```diff
@@ -53,6 +53,8 @@
         <appender-ref ref="STDOUT"/>
     </logger>
 
+    <logger name="migrations" level="INFO"/>
```

Comment:
We need the corresponding logger config added to `application-json-logger.xml` with the ability to override in conf